src/background.js

```diff
@@ -21,20 +21,80 @@
 
 // TODO: Encapsulate code and replace window.* calls throught code with pb.*
 
-var Utils = require("utils").Utils;
+var utils = require("utils");
 var DomainExceptions = require("domainExceptions").DomainExceptions;
 var HeuristicBlocking = require("heuristicblocking");
 var SocialWidgetLoader = require("socialwidgetloader");
 var pbStorage = require("storage");
 var webrequest = require("webrequest");
 var SocialWidgetList = SocialWidgetLoader.loadSocialWidgetsFromFile("src/socialwidgets.json");
 var Migrations = require("migrations").Migrations;
+var incognito = require("incognito");
 
-var pb = {
+
+// Display debug messages
+DEBUG = false,
```

Comment:
For consistency maybe this should be:

    var DEBUG = false;

    var constants = {
        // ...
    };